chroot-bin/croutoncycle

```diff
@@ -56,6 +56,16 @@ getname() {
     fi
 }
 
+# Nudges brightness down then restores old value, because when
+# chvt-ing back to cros, the brightness is not always correctly restored
+# (host dbus knows the correct value but hardware must be poked)
+nudgebrightness() {
+  local oldbrightness=`brightness get-untruncated`
+  brightness down &
```

Comment:
Don't fork here; wait for the commands to return.  Also, I suggest changing the brightness by a smaller offset than the default, so it's less obvious.  You'll also want to change it upwards if the volume is already at the minimum.  I suggest something like if oldbrightness < 50, nudge up. otherwise, nudge down.